bemlint/rules-description.md

```diff
@@ -0,0 +1,73 @@
+# one-block
```

Comment:
Would be great to improve the rule description and move each description to the separate file

Reply:
Still actual